firmware/library/L1_Drivers/onboardled.hpp

```diff
@@ -18,3 +20,63 @@ class OnBoardLedInterface {
            virtual void LedSetAll(uint8_t value)                     = 0;
            virtual uint8_t LedGetValues(void)                        = 0;
 };
+
+class OnBoardLed: public OnBoardLedInterface
+{
+ public:
+  // Initialization involes setting the appropriate pins to be GPIO, and
+  // then outputs. Once they are configured as such, they can be maniputlated
+  // as expected.
+  void Initialize(void) override
+    {
+    // This block will make sure all the GPIOs that were initialized in the
+    // last block will be set as outputs.
+    Led0.SetAsOutput();
+    // Led1.SetAsOutput();
+    // Led2.SetAsOutput();
+    // Led3.SetAsOutput();
```

Comment:
Why are these commented out?

Reply:
Ah, I realized I am doing a pull request on the wrong branch. I was trying to cut down on the number of errors. I should be pull requesting a different branch besides my experimental branch.